exercises/practice/dnd-character/dnd_character_test.rb

```diff
@@ -0,0 +1,52 @@
+require 'minitest/autorun'
+require_relative 'dnd_character'
+
+module Helpers
+  BASE_HITPOINTS = 10
+  MODIFIER_TO_CONSTITUTION = {
+    -4 => [3],
+    -3 => [4, 5],
+    -2 => [6, 7],
+    -1 => [8, 9],
+    0 => [10, 11],
+    1 => [12, 13],
+    2 => [14, 15],
+    3 => [16, 17],
+    4 => [18]
+  }.freeze
+
+  def attributes
+    %i[strength dexterity constitution intelligence wisdom charisma]
+  end
+end
+
+class DndCharacterTest < Minitest::Test
+  include Helpers
+
+  def test_hitpoints_correctly_calculated_from_constitution_modifier
+    character = DndCharacter.new
+    constitution_modifier = character.hitpoints - BASE_HITPOINTS
+
+    assert_includes MODIFIER_TO_CONSTITUTION[constitution_modifier], character.constitution
```

Comment:
We can add a message to the user if it fails that indicates what the result should have been.